packages/odyssey-storybook/src/components/Text/Text.mdx

```diff
@@ -0,0 +1,254 @@
+
+<Canvas>
+  <Story id="components-text-elements--address" />
+</Canvas>
+
+### blockquote
+
+> The `<blockquote>` HTML element indicates that the enclosed text is an extended quotation. Usually, this is rendered visually by indentation (see Notes for how to change it). A URL for the source of the quotation may be given using the cite attribute, while a text representation of the source can be given using the `<cite>` element. - [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)",
+
+<Canvas>
+  <Story id="components-text-elements--blockquote" />
+</Canvas>
+
+### cite
+
+> The HTML Citation element (`<cite>`) is used to describe a reference to a cited creative work, and must include the title of that work. - [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite)
+
+<Canvas>
+  <Story id="components-text-elements--cite" />
+</Canvas>
+
+### code
+
+> The `<code>` HTML element displays its contents styled in a fashion intended to indicate that the text is a short fragment of computer code. By default, the content text is displayed using the user agent's default monospace font. - [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code)
+
+<Canvas>
+  <Story id="components-text-elements--code" />
+</Canvas>
+
+### del
+
+> The `del` HTML element represents a range of text that has been deleted from a document. This can be used when rendering "track changes" or source code diff information, for example. The `ins` element can be used for the opposite purpose: to indicate text that has been added to the document. - [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del)
+
+#### Accessibility
+
+Many screen readers do not let users know of the presence of `del`. To fix this, you should consider using the ScreenReaderText component to prepend and append assistive text to the contents of the tag. In the above example, there are additional spaces before and after the text, this is intentional. Not adding these spaces will cause the content within the tag to run into the text within the tag.
+
+<Canvas>
+  <Story id="components-text-elements--del" />
+</Canvas>
+
+### dfn
+
+> The `dfn` is used to indicate the term being defined within the context of a definition phrase or sentence. The `p` element, the `dt`/`dd` pairing, or the `section` element which is the nearest ancestor of the `dfn` is considered to be the definition of the term. - [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Eldfnent/dfn)
+
+There are multiple, valid ways to use `dfn`. For the sake of usability, Odyssey recommends you follow one of two formats.
+
+For most terms, the content of `dfn` should be term you are defining.
+
+<Canvas>
+  <Story id="components-text-elements--dfn" />
+</Canvas>
+
+If you're referencing an acronym or abbreviation, you may also combine it with `<abbr>`:
+
+<Text as="p">
```

Comment:
We've run into our first major hurdle with CSF Stories + Arbitrary MDX:

> You can refer to existing stories (i.e. <Story id="...">) but cannot define new stories (i.e. <Story name="...">).

I'll open an issue for myself to test alternatives that won't set us back a sprint.